tests/core/opcodes/test_opcodes.py

```diff
@@ -618,8 +618,8 @@ def test_sstore(vm_class, code, gas_used, refund, original):
     computation = setup_computation(vm_class, CANONICAL_ADDRESS_B, decode_hex(code))
 
     computation.state.account_db.set_balance(CANONICAL_ADDRESS_B, 100000000000)
```

Comment:
With my comment above about moving to `state._account_db` , it *might* make sense to mirror these APIs as well...?